spec/support/contexts/default_rspec_language_config_context.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+# Inspired by:
+# https://github.com/rubocop-hq/rubocop-rspec/blob/cecc4b3f648002811c3956c1338431ecd4186aef/spec/shared/default_rspec_language_config_context.rb
+RSpec.shared_context "with default RSpec/Language config", :config do
```

Comment:
We had [an idea to extract this file into `lib/`](https://github.com/rubocop-hq/rubocop-rspec/pull/956/files#r515643915), so third-party gems can use it. A pull request is welcome if you decide to drop this file and use this shared context directly from `rubocop-rspec`.